python/cuml/multiclass/multiclass.py

```diff
@@ -115,6 +120,17 @@ def classes_(self):
     def n_classes_(self):
         return self.multiclass_estimator.n_classes_
 
+    def _input_to_host_array(self, X):
```

Comment:
This method does not use `self`. I would suggest to make it a module function instead. I think allowing for it to be replaced in inheritance introduces unnecessary complexity.